test/spec/extension.js

```diff
@@ -185,6 +225,45 @@ describe('extension - can copy', function() {
       expect(canCopyProperty).to.be.false;
     });
 
+    it('should allow on NoneEndEvents', function() {
```

Comment:
IMO we should not allow Input on NoneEndEvents? Do I miss something?

Reply:
Oops! You are correct. I did it correctly in the assertion `expect(canCopyProperty).to.be.false;` but the description of the test was wrong. Fixed it now.

Reply:
:+1: